.github/workflows/linux.yml

```diff
@@ -24,18 +25,14 @@ jobs:
             echo "invalid GO version"
             exit 1
           fi
-          echo "::set-output name=GO_VERSION::$go_version"
-      - uses: actions/setup-go@v2
+          echo "GO_VERSION=$go_version" >> $GITHUB_OUTPUT
```

Comment:
this is incorporating part of the change from https://github.com/aws/amazon-ecs-agent/pull/3487